editor/components/post-publish-panel/style.scss

```diff
@@ -59,3 +51,65 @@
 	color: $blue-medium-700;
 	text-decoration: underline;
 }
+
+.editor-post-publish-panel__prepublish {
+	padding: 16px;
+
+	.components-panel__body {
+		margin-left: -16px;
+		margin-right: -16px;
+		margin-bottom: -16px;
+		margin-top: 10px;
+		border: none;
+
+		.components-panel__body-toggle {
+			font-weight: normal;
+		}
+	}
+
+	.editor-post-visibility__dialog-legend {
+		display: none;
+	}
+}
+
+.post-publish-panel__postpublish .components-panel__body {
+	border-bottom: 1px solid $light-gray-500;
+	border-top: none;
+}
+
+.wp-core-ui .post-publish-panel__postpublish-buttons {
+	display: flex;
+	align-content: space-between;
+
+	> * {
```

Comment:
Why need this selector? It generally degrades performance a tiny bit.

Reply:
Since the "ClipboardButton" needs a wrapper, we can't target children buttons using a selector. I can replace it with something like `> .components__button, > span`, what do you think? I like the genericity of `> *` though?